script.js

```diff
@@ -0,0 +1,11 @@
+function toggleNav() {
+  const navUl = document.getElementById('nav-link');
+  const mainId = document.getElementById('main-logo');
+  const navrev = document.getElementById('reversed');
+  const closeHum = document.getElementById('close-hum');
+  navUl.classList.toggle('show');
+  mainId.classList.toggle('hide');
+  navrev.classList.toggle('reversed');
+  closeHum.classList.toggle('hide');
+}
+toggleNav();
```

Comment:
- [ ] Please as mentioned by the previous reviewer kindly make sure your design match as much as possible to the Figma design 

Your design
![Screenshot from 2022-07-19 12-13-32](https://user-images.githubusercontent.com/74501523/179730069-b8d6416f-477f-4fd4-9d52-c983c92533ab.png)

Figma design
![Screenshot from 2022-07-19 12-22-12](https://user-images.githubusercontent.com/74501523/179730141-aa6cc153-d1ee-4d01-8ba0-395c209d5904.png)

- [ ] Also when the user clicks (or taps) any option of the mobile menu, the mobile menu disappears but this is not the case for you when I click any option the mobile menu does not disappear, so please kindly fix it :upside_down_face: :upside_down_face:.